### ISSUE TITLE
Why "sample = sample.detach().requires_grad_()"

### ISSUE THREAD

**Issue by MrTornado24** [VITA-Group/NeuralLift-360#10]
Hey, thanks for your impressive work! As the title, I want to ask why you cut off the grad of sample in https://github.com/VITA-Group/NeuralLift-360/blob/942e4c10e4b802286b47c90bbee750724a42bf6e/nerf/sd_clipguide.py#L155

Doing so, the grad of the sample cannot be backward to NeRF. Is that reasonable?
